### ISSUE TITLE
[enhancement]: (fix) Clean right-click functionality

### ISSUE THREAD

**Issue by kenny-kvibe** [invoke-ai/InvokeAI#4947]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What should this feature add?

Fix for the right-click functionality, so that it doesn't sometimes open the browser context menu instead of the invokeai context menu.

Insert this into `<head>` of [`.venv\Lib\site-packages\invokeai\frontend\web\dist\index.html`](https://github.com/invoke-ai/InvokeAI/blob/main/invokeai/frontend/web/dist/index.html):
```html
<script type="text/javascript">'use strict';!function(){ window.oncontextmenu = () => false; }();</script>
 ```
 
This completely eliminates the browser's right click, atleast for me.

**Comment by psychedelicious** [comment]
We definitely want to allow the browser's right-click functionality in some places, so we aren't going to do a blanket no-op like this.

Where are the specific issues with right-click?

**Comment by kenny-kvibe** [comment]
@psychedelicious I use keyboard shortcuts for most operations but I understand people who don't.
It happened on Firefox in the right gallery pane, I've added that JS locally some days ago so I am not really sure where precisely.
Was just occasionally and always when I didn't need it to, so I mostly clicked something random each time because I was expecting the InvokeAI context menu. I did had a feelling that it's an event-timing thing - perhaps a fast/short-click with slight mouse-move could've possibly triggered it instead of ivokeai's, I didn't do any explicitly testing of it so I don't know the reason. Once I saw it triggered both of them, the browser's context overlaying the InvokeAI's context from one right-click, I disabled it.

You can use `event` in `oncontextmenu` and disable it for just the targeted `event.srcElement` element/s if you want.

**Comment by psychedelicious** [comment]
Maybe you were right-clicking between the images, but if you are getting the native context menu on top of the app context menu then I think there's more to the issue. 

Can you please undo the change you made so you can recreate the issue and upload a screen recording?

**Comment by kenny-kvibe** [comment]
I tried testing it & I figured out that my PC needs to use more resources and start lagging so that JS starts running slower than the browser and the browser-context triggers before the onmousedown/onclick event, this is it.
The recording wont be needed, I found the state in which it happens, when JS is Invoking and I have high CPU+RAM usage when I fast click around (which makes sense).

But I want to isolate the context menu from ever appearing, I use shift+right-click to force-open the browser's context menu, because this issue is not specific to your web UI, I ran across similar issues on different websites that have custom context menus like web-mails or web-IDEs, but it happens rarely.
I recently upgraded my PC so I'll need to observe if it even happens at low-resource usage.
